doc/source/cite.rst

```diff
@@ -12,9 +12,9 @@ or it can be read programatically as follows:
 
   import yaml
   import euphonic
-  from importlib_resources import open_text
+  from importlib_resources import files
 
-  with open_text(euphonic, 'CITATION.cff') as fp:
+  with open(files(euphonic).joinpath('CITATION.cff')) as fp:
```

Comment:
Could the `joinpath()` be a `/`? It looks like `files()` returns something Pathlib-ish.

Reply:
The [importlib_resources docs](https://importlib-resources.readthedocs.io/en/latest/using.html#packages-or-package-names) uses `joinpath` in their example, `/` works and is shorter (although as someone not used to that syntax its a little harder to read for me), but I guess there's no reason not to use it.

Reply:
I'm not entirely used to `/` for paths either, but I'd quite like to _get_ used to it :smile: